check-pull-request-status.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+from __future__ import print_function
+
+import sys
+
+from yaml import load
+from github3 import login
+
+if len(sys.argv) != 3:
+    print("USAGE: {} PULL_REQUEST_ID CREDENTIAL_STORE".format(sys.argv[0]))
+    exit(1)
+
+pull_request_id = int(sys.argv[1])
+credential_file = sys.argv[2]
+
+with open(credential_file) as credential_store:
+    credentials = load(credential_store)
+    user = credentials['github.com'][0]['user']
+    oauth_token = credentials['github.com'][0]['oauth_token']
+
+client = login(token=oauth_token)
+
+pull_request = client.pull_request(owner='openshift', repository='openshift-ansible', number=pull_request_id)
+pull_request_statuses = client._json(client._get(pull_request.statuses_url.replace('statuses', 'status')), 200)['statuses']
+
+blocking_statuses = [
+    'aos-ci-jenkins/OS_3.4_NOT_containerized',
+    'aos-ci-jenkins/OS_3.4_NOT_containerized_e2e_tests',
+    'aos-ci-jenkins/OS_3.4_containerized',
+    'aos-ci-jenkins/OS_3.4_containerized_e2e_tests',
+    'aos-ci-jenkins/OS_3.5_NOT_containerized',
+    'aos-ci-jenkins/OS_3.5_NOT_containerized_e2e_tests',
+    'aos-ci-jenkins/OS_3.5_containerized',
+    'aos-ci-jenkins/OS_3.5_containerized_e2e_tests',
+    'aos-ci-jenkins/OS_unit_tests',
+    'continuous-integration/travis-ci/pr',
+    'coverage/coveralls'
+]
```

Comment:
It will change as soon as we move master to 3.6 and 3.5. Also release-1.5 will only have 3.5 jobs, etc. Can we just check for all statuses green in an easy manner?